### DISCUSSION TITLE
Weird behavior with Callout

### DISCUSSION THREAD

**Discussion by jinapark202** [twilio-labs/paste#3486] [General] [answered]
When using Paste's callout component, it adds a bunch of extra space underneath the footer component on scroll. This bug doesn't happen with Evergreen's alert component. Any ideas why this might be happening?
![Screenshot 2023-09-13 at 4 42 56 PM](https://github.com/twilio-labs/paste/assets/87985289/575dae87-dab6-4d6b-94a7-098dc2b0c7eb)

**Comment by nkrantz** [comment]
Hey @jinapark202 - could you provide a [CodeSandbox](https://codesandbox.io/s/frosty-mopsa-mx3rkt) reproduction so we can take a look? Thanks for reporting

**Reply by jinapark202** [reply]
Not sure exactly how I'd be able to reproduce the situation in the CodeSandbox easily... Do you have access to the Segment app?

**Reply by serifluous** [reply]
We don't unfortunately. The Codesandbox reproduction helps us isolate the issue & helps us figure out if it's an issue with Paste or something in the Segment app affecting the component.

**Reply by jinapark202** [reply]
I added the code snippet that was causing the issue but I'm not sure I can exactly reproduce the issue there. Would it be helpful if I shared a video or something else?

**Reply by jinapark202** [reply]
Here's a video of the issue
https://www.loom.com/share/3e82b2f0302b4099a63e06e93b884fd4

**Reply by serifluous** [reply, marked as answer]
Thanks! I just messaged Cheryl on App Foundations, and she suggested you post in the #evergreen-paste-migration slack channel so folks can help debug there.
